### DISCUSSION TITLE
Import aus Nextcloud oder Mealie nicht möglich

### DISCUSSION THREAD

**Discussion by joachimarp** [TandoorRecipes/recipes#3407] [Q&A]
Nutze aktuell das Nextcloud Cookbook plane aber den Umstieg auf Tandoor. Habe Tandoor auf einer Linux VM im Docker installiert, klappt soweit auch. Import von einer Web-URL klappt auch prima. Nehme ich aber jetzt die Backup-Datei aus Nextcloud Cookbook oder auch aus Mealie und versuche die zu importieren, dann geschieht zunächst erstmal gar nichts. Irgendwann kommt dann der Fehler:

Failure
There was an error importing the recipe!

Sonst keine weiteren Information vom Programm, wieso. Hat jemand das Problem schon mal gehabt und hat einen Lösungsansatz? Docker, Docker-Composer und Tandoor sind in den aktuellen Versionen installiert. Der bereits bekannte Fehler mit der System-Seite ist auch vorhanden, weiß nicht ob das ggf. zusammenhängt.
![Screenshot 2024-11-22 150736](https://github.com/user-attachments/assets/b095ddbf-e40f-44f0-972d-ff81e2a4dcf6)

**Comment by vabene1111** [comment]
please provide logs and browser console outputs and maybe sample files for your imports, otherwise nobody can diagnose whats going wrong here

**Reply by joachimarp** [reply]
Ich glaube der Tipp mit der Browserkonsole war ganz nützlich, zumindest habe ich da den folgenden Fehler entdeckt.

`Failed to load resource: the server responded with a status of 413 (Request Entity Too Large)
POST http://192.168.177.6/api/import/ 413 (Request Entity Too Large)
(anonym) @ chunk-vendors.f3960bba6e8e.js:347
xhr @ chunk-vendors.f3960bba6e8e.js:347
ke @ chunk-vendors.f3960bba6e8e.js:347
_request @ chunk-vendors.f3960bba6e8e.js:347
request @ chunk-vendors.f3960bba6e8e.js:347
(anonym) @ chunk-vendors.f3960bba6e8e.js:347
(anonym) @ chunk-vendors.f3960bba6e8e.js:347
importAppRecipe @ import_view.cc5b92265060.js:1
click @ import_view.cc5b92265060.js:1
Or @ chunk-vendors.f3960bba6e8e.js:347
n @ chunk-vendors.f3960bba6e8e.js:347
push.70538.o._wrapper @ chunk-vendors.f3960bba6e8e.js:347
toolbarContentscript.js:113 {"message":"A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received"}`

**Reply by joachimarp** [reply]
Muss jetzt nur rausfinden, wie ich dem nginx beibringe die größeren Dateien zu akzeptieren

**Reply by vabene1111** [reply]
Eigentlch ist die standard config recht hoch, sonst mal im webserver (nginx) gucken

**Reply by joachimarp** [reply]
Es sind insgesamt 268 MB, die da im Stück importiert werden sollten. Ich habe in einem ersten Versuch ein kleineres Paket erstellt und importiert, hat soweit geklappt, nur das ich keine Bilder hatte.
